python_modules/dagster/dagster/_cli/api.py

```diff
@@ -79,6 +84,47 @@ def send_to_buffer(event):
                 sys.exit(return_code)
 
 
+def _is_isolated_run(dagster_run: DagsterRun) -> bool:
+    return dagster_run.tags.get("dagster/isolation") != "disabled"
+
+
+def _truthy_tag_value(dagster_run: DagsterRun, tag: str, default: str = "false") -> bool:
+    return dagster_run.tags.get(tag, default).casefold() in ["true", "1", "yes", "on"]
+
+
+def _should_start_metrics_thread(dagster_run: DagsterRun) -> bool:
+    return _truthy_tag_value(dagster_run, "dagster/run_metrics")
```

Comment:
I guess i would bias towards excluding falsey values vs. enumerating the possible set of truth-values, but not a dealbreaker

consider something like `dagster_run.tags.get(tag, "").tolower() not in {"0", "", "false"} `

Reply:
i agree that it isn't a particularly dangerous thing if i am a little bit more loose on validation